Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -18,6 +18,26 @@ import PackageLoading
 
 @testable import Build
 
+#if os(macOS)
+    let defaultTargetTriple: String =  "x86_64-apple-macosx10.10"
+#elseif os(Windows)
+    let defaultTargetTriple: String =  "x86_64-unknown-windows-msvc"
+#elseif os(Linux)
+    #if arch(x86_64)
+      let defaultTargetTriple: String = "x86_64-unknown-linux"
+    #elseif arch(i386)
+      let defaultTargetTriple: String = "i386-unknown-linux"
+    #elseif arch(powerpc64le)
+      let defaultTargetTriple: String = "ppc64le-unknown-linux"
+    #elseif arch(s390x)
+      let defaultTargetTriple: String = "s390x-unknown-linux"
+    #elseif arch(arm64)
+      let defaultTargetTriple: String = "arm64-unknown-linux"
+    #elseif arch(arm)
+      let defaultTargetTriple: String = "arm-unknown-linux"
```

Comment:
Please make this `armv7-unknown-linux-gnueabi` as this breaks exherbo otherwise.

Reply:
`armv7-unknown-linux-gnueabihf`, because that's what the rest of SwiftPM currently uses.